### PR TITLE
[13.0] [IMP] account_check_printing_report_base: Add letter format to A4 checks

### PR DESCRIPTION
**Pull request by CarlosRoca13**
@sergio-teruel @carlosdauden  This are the format improvements for promissory notes

cc @Tecnativa TT25536

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase to get possible green Travis

File: account_check_printing_report_base/i18n/es.po [hunk 1]
```diff
@@ -80,7 +80,7 @@ msgstr "Creado el"
 #. module: account_check_printing_report_base
 #: model_terms:ir.ui.view,arch_db:account_check_printing_report_base.report_check_base_a4
 msgid "Dear partner"
-msgstr "Querido"
+msgstr "Querido proveedor"
```
Comment (pedrobaeza) [inline account_check_printing_report_base/i18n/es.po]:
```suggestion
msgstr "Estimado proveedor"
```

File: account_check_printing_report_base/i18n/es.po [hunk 2]
```diff
@@ -103,26 +103,31 @@ msgstr "Descripción"
 #: model:ir.model.fields,field_description:account_check_printing_report_base.field_report_account_check_printing_report_base_report_check_base__display_name
 #: model:ir.model.fields,field_description:account_check_printing_report_base.field_report_account_check_printing_report_base_report_check_base_a4__display_name
 msgid "Display Name"
-msgstr "Nombre mostrado"
+msgstr "Mostrar Nombre"
```
Comment (pedrobaeza) [inline account_check_printing_report_base/i18n/es.po]: It was previously correct

```suggestion
msgstr "Nombre mostrado"
```

File: account_check_printing_report_base/i18n/es.po [hunk 3]
```diff
@@ -103,26 +103,31 @@ msgstr "Descripción"
...
 
 #. module: account_check_printing_report_base
 #: model_terms:ir.ui.view,arch_db:account_check_printing_report_base.report_check_base
 #: model_terms:ir.ui.view,arch_db:account_check_printing_report_base.report_check_base_a4
 msgid "Due Date"
-msgstr "Fecha vencimiento"
+msgstr "Fecha de Vencimiento"
```
Comment (pedrobaeza) [inline account_check_printing_report_base/i18n/es.po]: Upper letters must follow standard Spanish rules (no upper in the middle of a sentence except the known exceptions)

```suggestion
msgstr "Fecha de vencimiento"
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@pedrobaeza All changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**